disktools.py

```diff
@@ -635,7 +635,8 @@ def createPartition(self, id, sizeBytes=None, number=None, order=None, startByte
             'start': startSector,
             'size': sizeSectors,
             'id': id,
-            'active': active
+            'active': active,
+            'partlabel': label
```

Comment:
Leave a trailing comma after label.  It will make the next addition here cleaner.